_includes/sass/custom/_custom.scss

```diff
@@ -16,5 +16,6 @@
 @import "navigation";
 @import "sidenav";
 @import "filterselect";
+@import "commandpalette";
```

Comment:
👏  i'm psyched that this is command palette and not file finder, lets not limit it!